Rakefile

```diff
@@ -50,18 +61,25 @@ task :populate do
   p "Inserting example listings to the production database"
     connection = PG.connect(dbname: "airbnb")
 
-    connection.exec("INSERT INTO listings (name, description, price) VALUES('Hogwarts boys dorm', 'Magical dorm room for rent', 50);")
-    connection.exec("INSERT INTO listings (name, description, price) VALUES('4 Privet Drive', 'A roomy cosy space under staircase', 25);")
-    connection.exec("INSERT INTO listings (name, description, price) VALUES('Azkaban cell', 'No frills room available all year round!', 10);")
-    connection.exec("INSERT INTO listings (name, description, price) VALUES('Gryffindor House', 'A lovely warm space, would suit family of 300', 100);")
-    connection.exec("INSERT INTO listings (name, description, price) VALUES('Slytherin House', 'A welcoming space, would suit ex-murders', 50);")
-    connection.exec("INSERT INTO listings (name, description, price) VALUES('Hufflepuff House', 'A spacious place, great for all types of people', 75);")
-    connection.exec("INSERT INTO listings (name, description, price) VALUES('Ravenclaw House', 'A warm and cozy space, would suit family of 100', 75);")
+    connection.exec("INSERT INTO users (name, email, password) VALUES('jay', 'jay@test.com', 'jay123');")
+    connection.exec("INSERT INTO users (name, email, password) VALUES('jeff', 'jeff@test.com', 'jeff123');")
+    connection.exec("INSERT INTO users (name, email, password) VALUES('bibi', 'bibi@test.com', 'bibi123');")
+    connection.exec("INSERT INTO users (name, email, password) VALUES('kees', 'kees@test.com', 'kees123');")
+
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Hogwarts boys dorm', 'Magical dorm room for rent', 50, FALSE, FALSE, 1);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('4 Privet Drive', 'A roomy cosy space under staircase', 25, FALSE, FALSE, 2);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Azkaban cell', 'No frills room available all year round!', 10, FALSE, FALSE, 1);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Gryffindor House', 'A lovely warm space, would suit family of 300', 100, FALSE, FALSE, 3);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Slytherin House', 'A welcoming space, would suit ex-murders', 50, FALSE, FALSE, 1);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Hufflepuff House', 'A spacious place, great for all types of people', 75, FALSE, FALSE, 1);")
+    connection.exec("INSERT INTO listings (name, description, price, available, requested, user_id) VALUES('Ravenclaw House', 'A warm and cozy space, would suit family of 100', 75, FALSE, FALSE, 4);")
```

Comment:
Don't need the FALSE, FALSE  on each of these if we set default values (also remove available, requested from INSERT INTO).

Reply:
We want spaces to be available by default?